homeassistant/components/matrix/__init__.py

```diff
@@ -368,6 +370,28 @@ def _send_image(self, img, target_rooms):
                     ex.content,
                 )
 
+    def _send_video(self, vid, target_rooms):
```

Comment:
Given that the new code is almost identical to image sending, would it be possible to combine it with the existing command, and choose whether to call send_image or send_video based on the mime type?

Then you wouldn't need an extra command.

I ask more of an idea/discussion point than anything else.